examples/policygen/generated/forseti_policies/policies/templates/gcp_gke_enable_binauthz_v1.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Check to see if GKE stackdriver monitoring is enabled
+# https://cloud.google.com/monitoring/kubernetes-engine/
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-enable-binary-authorization-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPGKEEnableBinAuthzConstraintV1
+      validation:
+        openAPIV3Schema:
+          properties: {}
+  targets:
+    validation.gcp.forsetisecurity.org:
+      rego: | #INLINE("validator/gke_enable_binauthz.rego")
+            #
+            # Copyright 2019 Google LLC
```

Comment:
2021 - we should also update all the existing policy licenses.

Reply:
See the other comment.